src/stores/coreSettings.ts

```diff
@@ -528,5 +528,49 @@ export const CORE_SETTINGS: SettingParams[] = [
     type: 'boolean',
     defaultValue: true,
     versionAdded: '1.4.0'
+  },
+  {
+    id: 'Comfy.Pointer.ClickDrift',
```

Comment:
I am going to add a new top level category for all litegraph related features. The Comfy category now is very jammed.